### PR TITLE
Only announces direct buffs that are targeted ON the player

### PR DESCRIPTION
**Pull request by Nitrak**
Regular abilities such as Tranquility, Rallying Cry, Spirit Link Totem etc. is still announced.

destFlag 66833 is PLAYER (I could not find the correlating FINAL variable.

destFlag -2147483648 is abilities without a target (Integer.MIN) which we need to allow.

The Config change for saving the "onlySelf" variable is not as nice as I would like it, but it was the only way I could think of keeping it in the same group.

### CODE REVIEW

File: Config.lua [hunk 1]
```diff
@@ -57,21 +57,29 @@ local function createOptionsForClass(class)
 end
 
 local function setFilterValue(info, val)
-    local filter = filterValues[info]
```
Comment (nyy23) [inline Config.lua:60]: these setter & getter are used for setting the CLEU filter value. for setting a bool value in config, i would provide own set & get methods for your toggle

Comment (Nitrak) [reply Config.lua:60]: Fixed.

File: Locales/enUS.lua [hunk 2]
```diff
@@ -6,6 +6,7 @@ L["Discord"] = "Discord"
 L['PVE Voice Alert'] = "PVE Voice Alert"
 L["Profiles"] = "Profiles"
 L["My own abilities"] = "My own abilities"
+L["Buffs on player only"] = "Buffs on player only"
```
Comment (nyy23) [inline Locales/enUS.lua:9]: "Buffs on player only" might be to generic. We should clearly state, if you toggle this button you will only hear externals cast on you.

File: Config.lua [hunk 3]
```diff
@@ -239,7 +239,18 @@ local mainOptions = {
                             type = 'toggle',
                             name = L["Party member abilities"],
                             order = 2
-                        }
+                        },
+						onlyself = {
+							type = 'toggle',
+                            name = L["Buffs on player only"],
```
Comment (nyy23) [inline Config.lua:245]: Add some detailed description.

**Review by nyy23 — CHANGES_REQUESTED** [review]

File: Locales/enUS.lua (hunk 2 above)
Comment (Nitrak) [reply Locales/enUS.lua]: Updated name

File: Config.lua (hunk 3 above)
Comment (Nitrak) [reply Config.lua]: Added a clear description

**Review by nyy23 — APPROVED** [review]

**Pull request merged**